content/x-ray/x-ray-daemon.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Deploy X-Ray DaemonSet"
+date: 2018-11-177T08:30:11-07:00
+weight: 11
+draft: false
+---
+
+Now that we have modified the IAM role for the worker nodes to permit write operations to the X-Ray service, we are going to deploy the X-Ray [DaemonSet](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/) to the EKS cluster. The X-Ray daemon will be deployed to each worker node in the EKS cluster. For reference, see the [example implementation](https://github.com/rnzsgh/eks-workshop-x-ray-daemon) used in this module.
+
+The [AWS X-Ray SDKs](https://docs.aws.amazon.com/xray/index.html#lang/en_us) are used to instrument your microservices. When using the DaemonSet in the [example implementation](https://github.com/rnzsgh/eks-workshop-x-ray-daemon), you need to configure it to point to `xray-service.default:2000`.
+
+The following showcases how to configure the [X-Ray SDK for Go](https://docs.aws.amazon.com/xray/latest/devguide/xray-sdk-go.html). This is merely an example and not a required step in the workshop.
+
+```
+func init() {
+	xray.Configure(xray.Config{
+		DaemonAddr:     "xray-service.default:2000",
+		LogLevel:       "info",
+	})
+}
+```
+
+To deploy the X-Ray DaemonSet:
+
+```
+kubectl create -f https://raw.githubusercontent.com/rnzsgh/eks-workshop-x-ray-daemon/master/xray-k8s-daemonset.yaml
```

Comment:
Show the expected output so the user understands what is generated by submitting this.